src/engine/SpatialJoin.cpp

```diff
@@ -0,0 +1,295 @@
+#include "SpatialJoin.h"
+#include "util/AllocatorWithLimit.h"
+#include "util/MemorySize/MemorySize.h"
+#include "global/ValueId.h"
+#include "ValuesForTesting.h"
+#include "parser/ParsedQuery.h"
+#include "VariableToColumnMap.h"
+#include "engine/ExportQueryExecutionTrees.h"
+#include "util/GeoSparqlHelpers.h"
+
+SpatialJoin::SpatialJoin(QueryExecutionContext* qec, SparqlTriple triple,
```

Comment:
In the `.cpp` File, 
please add the empty `______________________________________` comments before each function. That way we see where they begin.